ext/rotoscope/stack.h

```diff
@@ -0,0 +1,35 @@
+#ifndef _INC_ROTOSCOPE_STACK_H_
+#define _INC_ROTOSCOPE_STACK_H_
+#include <stdbool.h>
+#include "tracepoint.h"
+
+#define UNKNOWN_STR "<UNKNOWN>"
+
+typedef struct rs_stack_frame_t
+{
+  const char *event;
+  const char *method_name;
+  const char *entity;
+  const char *method_level;
+  const char *filepath;
+  unsigned int lineno;
+  struct rs_stack_frame_t *caller;
+} rs_stack_frame_t;
```

Comment:
Yeah I considered this myself, but I didn't want to deeply couple the two data structures. That said, in my current implementation you need to pass in a `rs_tracepoint_t` to make a `rs_stack_frame_t` anyhow, so they're already pretty coupled. For simplicity, you're probably right 😸